db/migrate/20230128093911_create_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] data type for `bio` should be text and not string.  The difference between the two is how many characters you can put in these fields. `string` has less character limit as compared to `text. `